src/main/java/org/fundacion/jala/converter/models/parameter/VideoParameter.java

```diff
@@ -41,114 +42,127 @@ public VideoParameter(final String filePath, final String newOutputFormat, final
 
     /**
      * Sets the name of the output video.
-     * @param newOutputFormat String with the output format
+     *
+     * @param newOutputFormat is a String with the output format.
      */
     public void setOutputFormat(final String newOutputFormat) {
         this.outputFormat = newOutputFormat;
     }
 
     /**
      * Sets the resolution of the output video.
-     * @param newResolution String with the video resolution
+     *
+     * @param newResolution is a String with the video resolution
      */
     public void setResolution(final String newResolution) {
         this.resolution = newResolution;
     }
 
     /**
      * Sets the width of the output video.
-     * @param newWidth int with the video width
+     *
+     * @param newWidth is a int with the video width.
      */
     public void setWidth(final int newWidth) {
         this.width = newWidth;
     }
 
     /**
      * Sets the height of the output video.
-     * @param newHeight int with the video height
+     *
+     * @param newHeight is a int with the video height
      */
     public void setHeight(final int newHeight) {
         this.height = newHeight;
     }
 
     /**
      * Sets the audio of the output video.
-     * @param newAudio boolean that indicates if the video needs audio
+     *
+     * @param newAudio is a boolean that indicates if the video needs audio
```

Comment:
Dot is missing at the end of @param description.

Reply:
ok thaks Joel